app/assets/map.js

```
@@ -327,31 +364,133 @@ export function loadLevel(level, entryPoint = null) {
-			"You emerge into a clearing, the alien sun filtering through strange flora.",
-			"The ground is soft with moss, but the silence feels unnatural.",
-			"Twisted, alien trees loom over you like ancient sentinels.",
-		];
-		entryMessage = getRandomElement(forestMessages);
+	switch (planet.biome) {
+		case "Jungle":
+			const forestMessages = [
+				"You emerge into a clearing, the alien sun filtering through strange flora.",
+				"The ground is soft with moss, but the silence feels unnatural.",
+				"Twisted, alien trees loom over you like ancient sentinels.",
+			];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)